comprehensive-getting-started/Layer_Comprehensive_Getting_Started.ipynb

```diff
@@ -0,0 +1,883 @@
+        "All you need is to create a project readme markdown file by using your favourite text editor and put it into the working directory on your Google Colab notebook. Once you upload that file and run the ***layer.init*** function with the same project name again. [Layer](https://layer.ai/) will render and show it in the project main page similar to the screenshot below.\n",
+        "\n",
+        "Check out the page to learn more about Project cards on Layer:<br>\n",
+        "https://docs.app.layer.ai/docs/projects/card <br><br>\n",
+        "\n",
+        "\n",
+        "<img src=\" https://github.com/layerai/examples/blob/layer-comprehensive-demo/comprehensive-getting-started/images/Screenshot%202022-07-14%20at%2015.05.48.png?raw=true\" width=\"450\" height=\"600\" />\n",
+        "\n",
+        "\n",
+        "\n"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "odENGqP40Ae4"
+      },
+      "outputs": [],
+      "source": [
+        "# --- Run this cell once after you upload your README.md file into your project working directory ---\n",
+        "\n",
+        "layer.init(\"my-first-project\")"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "3JvG2yVNm_nS"
+      },
+      "source": [
+        "# **Step VII:** Run [Layer](https://layer.ai/) in local mode\n",
+        "---\n",
+        "\n",
+        "In order to use [Layer](https://layer.ai/) to store your ML metadata, you don't have use the ***layer.run()*** to run your functions remotely on the [Layer](https://layer.ai/) Cloud. You could also use your own resources to run your functions locally and still ask [Layer](https://layer.ai/) to store results returned from your functions. <br><br>\n",
+        "\n",
+        "In the cell below, we just call functions locally in their correct order and [Layer](https://layer.ai/) will still store generated results returned from those functions. Click on the respective links for each entity to see them on the [Layer](https://layer.ai/) Web UI."
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "b8-m2loGoseQ"
+      },
+      "outputs": [],
+      "source": [
+        "# --- Run your functions locally using your own resources and machines\n",
+        "\n",
+        "# my_first_dataset\n",
+        "create_dataset()\n",
+        "\n",
+        "# my_first_model\n",
+        "create_model()\n",
+        "\n",
+        "# predictions\n",
+        "generate_predictions()"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {},
+      "source": [
+        "# CONCLUSION\n",
+        "\n",
+        "Congratulations, you are done! You have completed the comprehensive Layer Demo by building an end-to-end ML pipeline.<br><br>\n",
```

Comment:
Might also be useful to add these 


To learn more about using layer, you can: 
- Join our [Slack Community ](https://bit.ly/layercommunityslack)
- Visit [Layer Examples Repo](https://github.com/layerai/examples) for more examples
- Browse [Trending Layer Projects](https://layer.ai) on our mainpage
- Check out [Layer Documentation](https://docs.app.layer.ai) to learn more